iOSCoreSample/iOSCoreSample/Utils/Constants.swift

```diff
@@ -1,15 +1,6 @@
-//
-//  Constants.swift
-//  iosApp
-//
-//  Created by Shaunak Jagtap on 10/08/22.
-//  Copyright © 2022 orgName. All rights reserved.
-//
-
-import UIKit
 import UIKit
 
-struct Constants {
+enum Constants {
```

Comment:
(doubt): Why did we change this to an `enum`? We don't have mutually exclusive constants in the file.